src/extensions/unitstable/Units/TableWidget.tsx

```diff
@@ -63,15 +82,17 @@ const HeaderRow: FunctionComponent<{columns: Column[], onColumnClick: (column: C
     )
 }
 
-const RowCheckbox = React.memo((props: {rowId: string, selected: boolean, onClick: (rowId: string) => void}) => {
-    const { rowId, selected, onClick } = props
+const RowCheckbox = React.memo((props: {rowId: string, selected: boolean, onClick: (rowId: string) => void, isDeselectAll?: boolean}) => {
+    const { rowId, selected, onClick, isDeselectAll } = props
     return (
         <Checkbox
             checked={selected}
+            indeterminate={isDeselectAll ? true : false}
```

Comment:
indeterminate=true means that there will be a minus sign in the checkbox, which i'm using to represent 'deselect all'